addons/web/static/src/views/view_button/view_header_button.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module */
+
+import { ViewButton } from "./view_button";
+
+export class ViewHeaderButton extends ViewButton {
```

Comment:
do we really need this extension of ViewButton? It looks like we could simply use ViewButton and directly give it the correct props (the buttonContext inside clickParams). In ViewButton, the props `record` should probably be renamed (in datapoint?) though. I think it only uses the datapoint API.